pkg/clients/pubsub.go

```diff
@@ -24,36 +24,64 @@ import (
 
 // PubSubMessenger implements the Messenger interface for Google Cloud pubsub.
 type PubSubMessenger struct {
-	projectID string
-	topicID   string
+	projectID  string
+	topicID    string
+	topicDLQID string
 
-	client *pubsub.Client
-	topic  *pubsub.Topic
+	client   *pubsub.Client
+	topic    *pubsub.Topic
+	topicDLQ *pubsub.Topic
```

Comment:
This might be a philosophical questions, but why have a messenger that can publish to two distinctly different topics instead of just creating two separate PubSubMessenger instances with different topic IDs?

Reply:
Good point, maybe this is the preferable approach. I came across [this API doc](https://pkg.go.dev/cloud.google.com/go/pubsub#Client.Topic), specifically the part that says "Avoid creating many Topic instances if you use them to publish."